include/boost/lockfree/queue.hpp

```diff
@@ -132,8 +132,8 @@ class queue
         node(void)
         {}
 
-        atomic<tagged_node_handle> next;
         T data;
+        atomic<tagged_node_handle> next;
```

Comment:
the order was actually on purpose to ensure alignment of the `atomic`

Reply:
reverted it back